src/main/thrift/parquet.thrift

```diff
@@ -269,36 +269,43 @@ enum GeometryEncoding {
 
 /**
  * Interpretation for edges of elements of a GEOMETRY logical type. In other
- * words, whether a point between two vertices should be interpolated in
- * its XY dimensions as if it were a Cartesian line connecting the two
- * vertices (planar) or the shortest spherical arc between the longitude
- * and latitude represented by the two vertices (spherical). This value
- * applies to all non-point geometry objects and is independent of the
- * coordinate reference system.
+ * words, whether points between two vertices should be interpolated in their
+ * XY dimensions as if they were on a straight line connecting the two vertices
+ * in Cartesian coordinates (linear) or on the shortest arc on a sphere between
+ * the longitude and latitude represented by the two vertices (pseudo geodesic).
+ * This value applies to all non-point geometry objects.
  *
- * Because most systems currently assume planar edges and do not support
- * spherical edges, planar should be used as the default value.
+ * The LINEAR value is independent of the coordinate reference system.
+ * The PSEUDO_GEODESIC value requires a coordinate reference system associated
+ * to a geodetic reference frame and uses spherical formulas even on ellipsoid,
+ * with geodetic latitudes used without conversion to authalic or other auxiliary
+ * latitudes.
+ *
+ * Because most systems currently assume linear edge interpolations and do
+ * not support geodesics, linear should be used as the default value.
  */
 enum Edges {
-  PLANAR = 0;
-  SPHERICAL = 1;
+  LINEAR = 0;
+  PSEUDO_GEODESIC = 1;
 }
 
 /**
  * Bounding box of geometries in the representation of min/max value pair of
- * coordinates from each axis when Edges is planar. Values of Z and M are omitted
- * for 2D geometries. When Edges is spherical, the bounding box is in the form of
+ * coordinates from each axis when the coordinate system is Cartesian.
+ * Values of Z and M are omitted for 2D geometries. When the coordinate system
+ * is spherical or ellipsoidal, the bounding box is in the form of
  * [westmost, eastmost, southmost, northmost], with necessary min/max values for
- * Z and M if needed.
+ * Z and M if needed. Note that the westmost value may be greater than the
+ * eastmost value if the bounding box crosses the anti-meridian.
  */
 struct BoundingBox {
-  /** Westmost value if edges = spherical **/
+  /** Westmost value if the coordinate system is spherical **/
```

Comment:
No problem, but then the word to change would be `spherical`. Maybe "if edges are geodesic"? (without "spherical" in order to mean "any kind of geodesic")?

Reply:
Changed _"if the coordinate system is spherical"_ by _"if edges are geodesics"_. I propose to use only "geodesics" instead of "pseudo geodesics" or "spherical geodesics" because that section is about min/max being interpreted as westmost/eastmost, which applies to any kind of geodesics. I can rename again if necessary.